users/views.py

```diff
@@ -0,0 +1,49 @@
+import jwt, datetime, requests
+
+from json.decoder import JSONDecodeError
+from django.views import View
+from django.http  import JsonResponse
+
+from users.models        import User
+from django.conf         import settings
+
+class KaKaoLoginView(View):
+    def get(self, request):
+        try:
+            kakao_token = request.headers.get('Authorization')
+            headers     = {'Authorization' : f'{kakao_token}'}
+            response    = requests.get('https://kapi.kakao.com/v2/user/me', headers=headers, timeout=3)
+            kakao_user  = response.json()
+
+            if response.status_code == -401:
+                return JsonResponse({'message' : 'Invalid token'}, status=401)
+
+        except KeyError: 
+            return JsonResponse({'message' : 'Key error'}, status=400)  
+
+        except JSONDecodeError: 
+            return JsonResponse({'message' : 'Json decode error'}, status=400)
+    
+        profile_image = kakao_user['kakao_account']['profile'].get('profile_image_url', None)
+        
+        login_user , state = User.objects.get_or_create(
```

Comment:
`get_or_create` 메소드 사용 시 생성된 경우에도 생성된 인스턴스가 반환됩니다. 따라서 `login_user`보다는 그냥 `user`로 적어주시는편이 더 좋을 것 같아요!
```suggestion
        user , state = User.objects.get_or_create(
```

Reply:
넵!